package.json

```diff
@@ -60,6 +72,7 @@
     "webpack-dev-server": "^3.5.1"
   },
   "dependencies": {
+    "enzyme": "^3.10.0",
```

Comment:
This should be in dev dependencies I think!

Reply:
Thank you @avpaul I have seen that and changed it.